$OpenBSD: patch-scripts_harch,v 1.1 2004/04/23 03:14:06 dons Exp $
Use arch as the canonical form for the processor name.
Makes substitution in PLISTs nicer.

--- script/harch.orig	Thu Apr 22 14:37:44 2004
+++ script/harch	Thu Apr 22 14:37:56 2004
@@ -48,8 +48,8 @@
 
 # Canonicalise equivalent processor families to one representative member.
 case $PROCESSOR in
-  i[3456789]86)	PROCESSOR=ix86;;
-  athlon)       PROCESSOR=ix86;;
+  i[3456789]86)	PROCESSOR=i386;;
+  athlon)       PROCESSOR=i386;;
   sun3*)	PROCESSOR=sun3;;
   sun4*)	PROCESSOR=sun4;;
   sparc*)	PROCESSOR=sparc;;
